sos/python/Sos.pyx

```diff
@@ -2942,8 +2942,13 @@ cdef class Attr(SosObject):
         return key
 
     def find(self, Key key):
-        cdef sos_index_t c_index = sos_attr_index(self.c_attr)
-        cdef sos_obj_t c_obj = sos_index_find(c_index, key.c_key)
```

Comment:
@narategithub we might want to guard against self.c_attr being NULL as well. Or does sos_attr_index check that for us?

Reply:
Hi @narategithub, I didn't see a response to my last comment.

Reply:
@tom95858 oops. Sorry I let this slipped. `self.c_attr` is guaranteed to be NOT NULL. If `self.c_attr` is NULL, the `Attr.__init__()` will raise an exception:

https://github.com/ovis-hpc/sos/blob/83c8f8462f08cfb238be550b85bb23e4149d35de/sos/python/Sos.pyx#L2797-L2805)